Sources/KituraWebSocket/WebSocketConnection.swift

```diff
@@ -244,6 +259,16 @@ extension WebSocketConnection: ChannelInboundHandler {
         }
     }
 
+    public func userInboundEventTriggered(context: ChannelHandlerContext, event: Any) {
+        guard event is IdleStateHandler.IdleStateEvent else { return }
```

Comment:
usually the rules are that if you don't handle a user event, you should forward it (as opposed to dropping it). This might be the last handler in the pipeline so it might be fine...

Reply:
Yes, this is the last inbound handler in the pipeline.

Reply:
Added a change to propagate events that aren't `IdleStateEvent`s